packages/fxa-payments-server/src/components/NewUserEmailForm/index.scss

```diff
@@ -1,27 +1,6 @@
 @import '../../../../fxa-content-server/app/styles/variables';
 
 .new-user-email-form {
```

Comment:
I added a note to the `fields` ticket as a reminder that `NewUserEmailForm` has a style for `fields`.

Reply:
Thank you!